backend/app/src/main/java/com/moh/phlat/backend/service/RowStatusService.java

```diff
@@ -5,6 +5,21 @@
 import java.util.List;
 
 public interface RowStatusService {
+    String UPLOAD_COMPLETED = "UPLOAD_COMPLETED";
+    String UPLOAD_ERROR = "UPLOAD_ERROR";
+    String DO_NOT_LOAD = "DO_NOT_LOAD";
+    String INITIAL = "INITIAL";
+    String VALID = "VALID";
+    String INVALID = "INVALID";
+    String APPROVED = "APPROVED";
+    String COMPLETED ="COMPLETED";
+    String PRE_VALIDATION_IN_PROGRESS ="PRE-VALIDATION_IN_PROGRESS";
```

Comment:
some constants have dash in them. Please check

Reply:
Debugging statements have been removed,

Reply:
the constants are still having the dashes in the names. Any reason why it was resolved?